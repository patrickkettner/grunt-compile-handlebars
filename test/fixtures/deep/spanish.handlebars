<h1>{{saludo}}{{puntuacion}} {{ubicacion}}</h1>{{> shared/foo}}
{{super_helper}}