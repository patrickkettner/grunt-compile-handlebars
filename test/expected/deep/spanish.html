<h1>hola, mundo</h1>foo partial
super helper!